src/Cms/Helpers.php

```diff
@@ -117,6 +117,12 @@ public static function handleErrors(
 	) {
 		$override = null;
 
+		// backup current locale
+		$locale = setlocale(LC_MESSAGES, 0);
+
+		// set locale to C to avoid locale errors
```

Comment:
`Suggestion` to make clearer why all of this is needed:

```suggestion
		// set locale to C so that errors and warning messages are
		// printed in English for robust comparisons in the handler
```